index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sameed - Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap" rel="stylesheet">
+</head>
+
+
+
+<body>
+
+    <header>
+        <nav>
+            <div class="logo-container">
+                <img src="images for portfolio/logo.jpg" alt="" class="logo">
+                <a class="logo-text">Sameed Mohsin</a>
+            </div>
+
+            <div class="menu">
+                <img src="./Images for portfolio/Hamburger.svg" alt="" class="hamburger" width="50px" height="50p">
+            </div>
+        </nav>
+    </header>
+
+    <section>
+        <div>
+            <h1 class = "top-heading"> I'm Sameed</h1>
+        </div>
+        <div class="my-description">
+            <p>I am a web developer.</p>
+            <p>You can count on me for building, developing, and maintaining your websites.</p>
+        </div>
+
+    </div>
+    
+</section>
+
+<footer>
```

Comment:
This doesn't come in a `footer` tag, you have read the material again in depth. You can use a simple `div` here. Kindly read about sematic tags [here](https://www.w3schools.com/html/html5_semantic_elements.asp)